modules/solid_mechanics/src/materials/IsotropicPowerLawHardening.C

```diff
@@ -65,7 +65,7 @@ IsotropicPowerLawHardening::computeStressInitialize(unsigned qp, Real effectiveT
 }
 
 Real
-IsotropicPowerLawHardening::computeHardening( unsigned qp, Real scalar )
+IsotropicPowerLawHardening::computeHardening( unsigned /*qp*/, Real scalar )
 {
   Real stress = _effectiveTrialStress - 3.0 * _shear_modulus * scalar;
   Real slope = std::pow(stress, 1.0/_n -1.0)/ _n/ std::pow(_K, 1.0/_n) - 1.0/_youngs_modulus;
```

Comment:
The changes to IsotropicPowerLawHardening look fine. The yield stress and the hardening slope do not depend on qp in this case. Thanks!